src/snowflake/connector/network.py

```diff
@@ -20,8 +20,10 @@
 import requests
 from requests.adapters import HTTPAdapter
 from requests.auth import AuthBase
-from requests.exceptions import ConnectionError, ConnectTimeout, ReadTimeout, SSLError
+from requests.exceptions import ConnectionError, ConnectTimeout, InvalidProxyURL, ReadTimeout, SSLError
 from requests.packages.urllib3.exceptions import ProtocolError, ReadTimeoutError
+from requests.utils import prepend_scheme_if_needed, select_proxy
+from urllib3.util.url import parse_url
```

Comment:
These import statements should be updated to the vendored versions

Reply:
I will merge the latest master to this PR. But thinking more about this - do we want to restrict vendored request to stuff which we foresee using OCSP?